core/src/main/scala/kafka/common/Topic.scala

```diff
@@ -19,7 +19,7 @@ package kafka.common
 
 import util.matching.Regex
 import kafka.coordinator.GroupCoordinator
-
+import org.apache.kafka.common.errors
```

Comment:
Nit: Given that only InvalidTopicException is used in this file, can we just import that instead of the package?